chia/server/rate_limits.py

```diff
@@ -92,6 +92,9 @@ class RLSettings:
     ProtocolMessageTypes.request_header_blocks: RLSettings(500, 100),
     ProtocolMessageTypes.reject_header_blocks: RLSettings(100, 100),
     ProtocolMessageTypes.respond_header_blocks: RLSettings(500, 2 * 1024 * 1024, 100 * 1024 * 1024),
+    ProtocolMessageTypes.request_block_headers: RLSettings(500, 100),
+    ProtocolMessageTypes.reject_block_headers: RLSettings(100, 100),
+    ProtocolMessageTypes.respond_block_headers: RLSettings(500, 2 * 1024 * 1024, 100 * 1024 * 1024),
```

Comment:
this probably needs to be adjusted

Reply:
I take it this means you can download at most 100 MiB of blocks from a peer, is that right?
@mariano54

Reply:
Correct

Reply:
We should increase the number of request per minute, wallets can make many of these requests